src/Simplex/Chat/Types.hs

```diff
@@ -184,6 +190,7 @@ data GroupInfo = GroupInfo
     localDisplayName :: GroupName,
     groupProfile :: GroupProfile,
     membership :: GroupMember,
+    membershipIncognito :: Bool,
```

Comment:
it would probably follow from the fact the memberProfile and contactProfileId are different, so `membershipIncognito` is probably not needed - am I right?